packages/datatrak-web-server/src/routes/SurveyRoute.ts

```diff
@@ -15,7 +15,7 @@ export type SurveyRequest = Request<
   DatatrakWebSurveyRequest.ReqQuery
 >;
 
-const DEFAULT_FIELDS = ['name', 'code', 'id', 'can_repeat', 'survey_group.name', 'surveyQuestions'];
+const DEFAULT_FIELDS = ['name', 'code', 'id', 'can_repeat', 'survey_group.name', 'project_id'];
```

Comment:
```suggestion
const DEFAULT_FIELDS = ['name', 'code', 'id', 'can_repeat', 'survey_group.name', 'project_id', 'surveyQuestions'];
```
I'm not sure if this was a merge issue, but I believe we still want to the `surveyQuestions` in this route, right?

Reply:
Yeah that wasn't my change I don't think, now that you say that